pydeseq2/dds.py

```diff
@@ -542,49 +525,42 @@ def fit_LFC(self):
                 batch_size=self.batch_size,
             )(
                 delayed(irls_solver)(
-                    counts=counts_nonzero[:, i],
-                    size_factors=size_factors,
+                    counts=self.X[:, i],
+                    size_factors=self.obsm["size_factors"],
                     design_matrix=X,
-                    disp=disps[i],
+                    disp=self.varm["dispersions"][i],
                     min_mu=self.min_mu,
                     beta_tol=self.beta_tol,
                 )
-                for i in range(num_genes)
+                for i in self.non_zero_idx
             )
         end = time.time()
         print(f"... done in {end-start:.2f} seconds.\n")
 
-        MAP_lfcs_, mu_, hat_diagonals_, converged_ = zip(*res)
+        MLE_lfcs_, mu_, hat_diagonals_, converged_ = zip(*res)
```

Comment:
why this change in terminology?

Reply:
`MAP_lfcs` should be post-shrinkage LFCs. The previous name was a mistake

Reply:
ok thanks!